Automattic/abacus
# Fix experiment draft min diff precision

Address: https://github.com/Automattic/experimentation-platform/issues/971

This fixes the precision issue for min diff values during when copying the experiment design in `/experiments/{experiment_id}/setup`.

## How has this been tested?






- Manual testing with production data (locally)
- Manual testing with mock data (locally or on Vercel)
- Additional manual testing instructions (please specify):

Go to an experiment setup page and click `Copy experiment design`. Paste the contents into p2 draft. The new decimals should be fixed for conversion metrics.

I couldn't get this to work when testing locally, but I checked the values are being properly scaled with `console.log()`.
